ddsrecorder/src/cpp/command_receiver/CommandReceiver.cpp

```diff
@@ -155,6 +156,11 @@ bool CommandReceiver::init()
     // Enable type information sending
     pqos.wire_protocol().builtin.typelookup_config.use_server = true;
 
+    // Set Intraprocess OFF
```

Comment:
I would add a comment on why this is required, and stating this is a temporal workaround.

Reply:
```suggestion
    // Set Intraprocess OFF
    // WORKAROUND: This is a temporal solution to fix a potential deadlock in the communication between a recorder and its corresponding command receiver (both being in the same DDS domain). More precisely, the deadlock affects the current implementation of TypeLookupService module with intraprocess communication.
```